website/src/pages/admin/manage_user/[id].tsx

```diff
@@ -143,10 +143,13 @@ const ManageUser = ({ user }: InferGetServerSidePropsType<typeof getServerSidePr
 const UserMessageTable = ({ id }: { id: User["id"] }) => {
   const { pagination, toNextPage, toPreviousPage } = useCursorPagination();
   const { data, error, isLoading } = useSWRImmutable<FetchUserMessagesCursorResponse>(
-    `/api/admin/user_messages?user=${id}&cursor=${encodeURIComponent(pagination.cursor)}&direction=${
-      pagination.direction
-    }`,
-    get,
+    "/api/admin/user_messages",
```

Comment:
Please revert these changes

Reply:
I will if you insist, but this is the correct way that it's supposed to work.  This is how swr is supposed to be used.  Can we can discuss on discord please?

Reply:
Sorry, here it needs to be reverted

Reply:
Done